mr_adc_integrals.py

```diff
@@ -44,17 +44,16 @@ def transform_integrals_1e(mr_adc):
 
     print("Time for transforming 1e integrals:                %f sec\n" % (time.time() - start_time))
 
-def transform_2e_phys_incore(interface, mo_1, mo_2, mo_3, mo_4):
-    'Two-electron integral transformation in Physicists notation'
+def transform_2e_chem_incore(interface, mo_1, mo_2, mo_3, mo_4):
+    'Two-electron integral transformation in Chemists notation'
 
     nmo_1 = mo_1.shape[1]
     nmo_2 = mo_2.shape[1]
     nmo_3 = mo_3.shape[1]
     nmo_4 = mo_4.shape[1]
 
-    v2e = interface.transform_2e_chem_incore(interface.v2e_ao, (mo_1, mo_3, mo_2, mo_4), compact=False)
-    v2e = v2e.reshape(nmo_1, nmo_3, nmo_2, nmo_4)
-    v2e = v2e.transpose(0,2,1,3)
+    v2e = interface.transform_2e_chem_incore(interface.v2e_ao, (mo_1, mo_2, mo_3, mo_4), compact=False)
+    v2e = v2e.reshape(nmo_1, nmo_2, nmo_3, nmo_4)
```

Comment:
Is this a contiguous array?

Reply:
Yes, they came from the PySCF interface as contiguous arrays. I tested them.